File: aoj/volume100/csharp/A10007.cs
using System;

class AOJ
{
	static void Main()
	{
		while (true)
		{
			var input = Console.ReadLine().Split();
			var a = int.Parse(input[0]);
			var b = int.Parse(input[1]);
			if (a == 0 && b == 0)
				break;

			if (a < b)
				Console.WriteLine("{0} {1}", a, b);
			else
				Console.WriteLine("{0} {1}", b, a);
		}
	}
}

File: aoj/volume100/scala/A10007.scala
object Main {
  def main(args: Array[String]) {
    loop
  }

  def loop {
    val ls = (scala.io.StdIn.readLine split " ").map(_.toInt)
    if (ls(0) == 0 && ls(1) == 0) {
      return
    }

    val sorted = ls.sorted
    println(s"${sorted(0)} ${sorted(1)}")
    loop
  }
}
